code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -89,7 +89,7 @@
 /obj/machinery/atmospherics/Initialize(mapload)
 	if(mapload && name != initial(name))
 		override_naming = TRUE
-	return ..()	
```

Comment:
You summoned us for these

Reply:
skill issue

Reply:
first time?

Reply:
I'm gonna downvote the pr and there is nothing you can do about it!!!!!!?!

Reply:
oh yeah so will i

Reply:
I hate you two

Reply:
:trollface: